index.md

```diff
@@ -32,7 +32,7 @@ So far, none.
 
 ### 2.3 Definitions
 
- * **AV1 access unit**: a Decodable Frame Group as defined in Annex E of AV1 Bitstream & Decoding Process Specification.
+ * **Decodage Frame Group (DFG)** : a Decodable Frame Group as defined in Annex E of the AV1 Bitstream & Decoding Process Specification.
```

Comment:
Probably want a better word than "Decodage"

Reply:
Correct.

Reply:
It's decodable